### PR TITLE
support run level0 case in cypress

### PR DESCRIPTION
**Pull request by yanpzhan**
@yapei pls help to review. Thanks!

### CODE REVIEW

**Comment by XiyunZhao** [comment]
Link with ticket https://issues.redhat.com/browse/OCPQE-20721

**Comment by yapei** [comment]
@liangxia I confirmed with @chengzhang1016 that we can temporarily add cypress tests in the automated release chain to see how it will be running, we can still disable if they are consistently unstable

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-fips-f1

File: ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml [hunk 1]
```diff
@@ -8,7 +8,8 @@ chain:
   - ref: cucushift-pre
   - ref: openshift-extended-test
   - ref: cucushift-e2e
+  - ref: openshift-extended-web-tests
```
Comment (liangxia) [inline ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml:11]: We should not add cypress test there, as this chain is also used by vsphere tests which are supposed not to run cypress tests.

Comment (yanpzhan) [reply ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml:11]: @liangxia Where do you suggest to add cypress test for auto release job?

Comment (yanpzhan) [reply ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml:11]: add directly in ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release.yaml ?

Comment (liangxia) [reply ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml:11]: Let's make sure cypress can run level0 test cases and make sure those level0 test cases are stable first.

Comment (yanpzhan) [reply ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml:11]: We only have several level0 tag cases. Remove the ref step in current pr, could add the web test step after we consider how to adapt it to different auto release jobs including vsphere.

**Comment by yanpzhan** [comment]
@yapei @liangxia Only keep the tag check in command script in this pr. Let's include cypress test in separate pr after we decide to add in which job/how to add.

**Pull request merged**